src/compas/datastructures/_mixins/filters.py

```diff
@@ -78,6 +90,62 @@ def vertices_where(self, conditions, data=False):
                 else:
                     yield key
 
+    def vertices_where_predicate(self, predicate, data=False):
+        """Get vertices for which a certain condition or set of conditions is true using a lambda function.
+
+        Parameters
+        ----------
+        predicate : function
+            The condition you want to evaluate.
```

Comment:
explain the expected call signature of the lambda function

Reply:
Done, but ``invoke docs`` gives me some errors and I cannot check if it displays correctly.

Reply:
use ``callable`` instead of ``function``

Reply:
I've change them.